<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1443</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LaTeX Editor</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1231</width>
      <height>631</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1221</width>
       <height>581</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Title Page</string>
      </attribute>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>1201</width>
         <height>521</height>
        </rect>
       </property>
      </widget>
      <widget class="QFontComboBox" name="fontComboBox">
       <property name="geometry">
        <rect>
         <x>980</x>
         <y>0</y>
         <width>223</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Packages</string>
      </attribute>
      <widget class="QTextEdit" name="textEdit_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>1201</width>
         <height>521</height>
        </rect>
       </property>
      </widget>
      <widget class="QFontComboBox" name="fontComboBox_2">
       <property name="geometry">
        <rect>
         <x>990</x>
         <y>0</y>
         <width>223</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Commands</string>
      </attribute>
      <widget class="QTextEdit" name="textEdit_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>1201</width>
         <height>521</height>
        </rect>
       </property>
      </widget>
      <widget class="QFontComboBox" name="fontComboBox_3">
       <property name="geometry">
        <rect>
         <x>980</x>
         <y>0</y>
         <width>223</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Contents</string>
      </attribute>
      <widget class="QTextEdit" name="textEdit_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>1201</width>
         <height>521</height>
        </rect>
       </property>
      </widget>
      <widget class="QFontComboBox" name="fontComboBox_4">
       <property name="geometry">
        <rect>
         <x>980</x>
         <y>0</y>
         <width>223</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>590</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>590</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>590</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Show PDF</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>310</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Predefined Templates</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sonny</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Lenny</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Glenn</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Conny</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Rejne</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bjarne</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bjornstrup</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>20</y>
      <width>191</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Titlepage</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>93</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1443</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="actionOpen_File"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew_File">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
